snaps-sidebar.js

```diff
@@ -26,24 +26,24 @@ const sidebar = {
     },
     {
       type: "category",
-      label: "How to",
-      link: { type: "generated-index", slug: "/how-to" },
+      label: "Learn",
```

Comment:
For Snaps developers, we wanted to emphasize both the learning material (since most coming to the docs are new to the system), and feature/use case guides (to highlight specific functionalities that Snaps enable), which is why those sections are higher up in the sidebar.